packages/medusa-cli/src/commands/new.ts

```diff
@@ -553,16 +556,17 @@ medusa new ${rootPath} [url-to-starter]
 
     if (/medusa-starter/gi.test(rootPath) && isStarterAUrl) {
       reporter.panic({
-        id: `10000`,
+        id: PanicId.InvalidProjectName,
```

Comment:
comment: The `id` here and on line 569 is different on `develop`, but you seem to use the same here. Just want to be sure this is intentional

Reply:
Yes I just simplified since both ids returned the same error message, i thought reducing to a single id with the same error message was equivalent. Wdyt?

Reply:
Yes sounds good. Only highlighted it to make sure it was intentional 👍